Source/Core/DolphinQt2/QtUtils/AddConnectedAction.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QAction>
+#include <QObject>
+#include <QString>
+#include <QWidget>
+
+// Qt5 added a new, compile-time way to connect to signals (passing functors or member function
+// pointers directly), but QMenu's addAction method didn't get equivalent functionality until Qt
+// 5.6. These functions are meant to be drop-in replacements for the new QMenu::addAction overrides
+// until Dolphin requires at least Qt 5.6 to build.
```

Comment:
I'm more thinking that it doesn't even compile on older versions because we have lines [like this](https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/DolphinQt2/Main.cpp#L26). Wouldn't `Qt::AA_EnableHighDpiScaling` be undefined in Qt <5.6 and refuse to compile?